releases/net471/KnownIssues/623552-BCL Higher assembly versions that 4.0.0.0 for System.IO.Compression.ZipFile cannot be loaded without a binding redirect.md

```diff
@@ -0,0 +1,36 @@
+# Higher assembly versions that 4.0.0.0 for System.IO.Compression.ZipFile cannot be loaded without a binding redirect
+
+## Symptoms
+
+When building an application that targets .NET Framework 4.7.1, referencing a .NET Standard library and use types from the `ZipFile` assembly you will get a runtime error similar to this one:
```

Comment:
Better; referencing a .NET Standard library and use types from the `ZipFile` assembly you will get a runtime error similar to this one: --> if you reference a .NET Standard library that uses types from the `ZipFile` assembly, a runtime error like the following results: